.github/CODEOWNERS

```diff
@@ -232,7 +232,7 @@
 /cmd/system-probe/modules/tcp_queue_tracer* @DataDog/container-integrations
 /cmd/system-probe/modules/traceroute*   @DataDog/network-device-monitoring @Datadog/Networks
 /cmd/system-probe/modules/ping*         @DataDog/ndm-core
-/cmd/system-probe/modules/language_detection* @DataDog/container-intake @DataDog/universal-service-monitoring
+/cmd/system-probe/modules/language_detection* @DataDog/container-intake @DataDog/agent-discovery
```

Comment:
This and the other language detection path below should probably not have USM removed since USM was the owner from back when the package was added. Maybe you could _add_ agent-discovery instead?